programmers-coding-test/sangjun/Implement/게임개발.js

```diff
@@ -0,0 +1,60 @@
+const rl = require("readline-sync");
+
+const N = rl.question("n*n 맵생성 :\n>> ").split(" ");
+const characterLocation = rl.question("캐릭터위치 바라보는방향입력 (ex 1 1 0(북)) :\n>> ").split(" ");
```

Comment:
밑에서 다시 분해할당하지 말고 애초에 입력하고 바로 해주는 것이 좋겠다.
```suggestion
const [x, y, direction] = rl.question("캐릭터위치 바라보는방향입력 (ex 1 1 0(북)) :\n>> ").split(" ");
```